attestation-service/docs/parsed_claims.md

```diff
@@ -92,3 +92,27 @@ The claim inherit the fields from the SEV-SNP claim with and additional `tpm` hi
 - `tpm.pcr{01,..,n}`: SHA256 PCR registers for the TEE's vTPM quote.
 
 Note: The TD Report and TD Quote are fetched during early boot in this TEE. Kernel, Initrd and rootfs are measured into the vTPM's registers.
+
+## AMD SEV-SNP
+
+- `measurement` Launch Digest covering initial guest memory
+- `platform_smt_enabled`:  Whether Simultaneous Multithreading is enabled on the system
+- `platform_tsme_enabled`: Whether Transparent SME is enabled on the system
+- `policy_abi_major`: Minimum ABI major version allowed for guest
+- `policy_abi_minor`: Minimum ABI minor version allowed for guest
+- `policy_debug_allowed`: Whether SNP debug features are allowed for guest
+- `policy_migrate_ma`: Whether migration agent can be connected to guest
+- `policy_single_socket`: Whether guest can be activated only on one socket
+- `policy_smt_allowed`: Whether guest can run on a system with SMT enabled
+- `reported_tcb_bootloader`: Reported SVN of ASP bootloader
+- `reported_tcb_microcode`: Reported microcode version
+- `reported_tcb_snp`: Reported SVN of SNP Firmware
+- `reported_tcb_tee`: Reported SVN of ASP OS
```

Comment:
all claims above would be add a prefix `snp.` due to the function https://github.com/confidential-containers/trustee/blob/main/attestation-service/attestation-service/src/utils.rs#L39